share/models/change.py

```diff
@@ -242,6 +243,7 @@ def _resolve_change(self):
                 else:
                     from share.models.base import ExtraData
                     change[k] = ExtraData()
+                change[k].change = self
```

Comment:
Ignore me, I skipped that part of elementary school where they teach you to read.